src/docfx/Errors.cs

```diff
@@ -28,20 +28,17 @@ public static Error InvalidLocale(string locale)
         public static Error CircularReference<T>(T filePath, IEnumerable<T> dependencyChain)
             => new Error(ErrorLevel.Error, "circular-reference", $"Found circular reference: {string.Join(" --> ", dependencyChain.Select(file => $"'{file}'"))} --> '{filePath}'", filePath.ToString());
 
-        public static Error UserProfileCacheNotFound(string userProfilePath)
-            => new Error(ErrorLevel.Error, "user-profile-cache-not-found", $"Cannot find user profile cache at '{userProfilePath}'");
+        public static Error UrlRestorePathNotFound(string url)
+            => new Error(ErrorLevel.Error, "url-restore-path-not-found", $"The restore path of url `{url}` can't be found, please make sure the `restore` command was executed");
 
         public static Error InvalidUserProfileCache(string userProfileCache, Exception ex)
             => new Error(ErrorLevel.Error, "invalid-user-profile-cache", ex.Message, userProfileCache);
 
-        public static Error GitCommitsTimeNotFound(string gitCommitsTimePath)
-            => new Error(ErrorLevel.Error, "git-commits-time-not-found", $"Cannot find git commits time at '{gitCommitsTimePath}'");
-
         public static Error InvalidGitCommitsTime(string gitCommitsTimePath, Exception ex)
             => new Error(ErrorLevel.Error, "invalid-git-commits-time", ex.Message, gitCommitsTimePath);
 
         public static Error DependenyRepoNotFound(string dependenyRepoHref)
```

Comment:
>DependenyRepoNotFound [](start = 28, length = 21)

maybe merge these 2 to a single error `restore-not-run` ?

Reply:
would like to tell user more clear error message 

---
In reply to: [202249636](https://github.com/dotnet/docfx/pull/3065#discussion_r202249636) [](ancestors = 202249636)